packages/block-library/src/media-text/edit.js

```diff
@@ -37,6 +37,7 @@ const ALLOWED_BLOCKS = [ 'core/button', 'core/paragraph', 'core/heading', 'core/
 const TEMPLATE = [
 	[ 'core/paragraph', { fontSize: 'large', placeholder: _x( 'Content…', 'content placeholder' ) } ],
 ];
+const applyWidthConstraints = ( width ) => Math.max( 15, Math.min( width, 85 ) );
```

Comment:
I think before the merge we should move these values to constants at the top of the file so they can be easily changed.